gap/Magic.gd

```diff
@@ -296,6 +289,17 @@
 #!             <Br/>
 #!             <E>Default value: <C>[ ".", "gap", "lib", "examples", "examples/doc" ]</C>.</E>
 #!         </Item>
+#!
+#!         <Mark><A>LaTeXOptions</A></Mark>
+#!         <Item>
+#!             Must be a record with entries which can be understood by
+#!             <Ref Func='SetGapDocLaTeXOptions' BookName='gapdoc'/>. Each entry can be a
+#!             string, which will be given to &GAPDoc; directly, or a list containing of
+#!             two entries: The first one must be the string "file", the second one a
+#!             filename. This file will be read and then its content is passed to &GAPDoc;
+#!             as option with the name of the entry.
```

Comment:
Wait, what?!? I never knew we supported that... else I'd have loudly complained to @sebasguts ... :-). This seems to be a pointless complication, as one can simply use `StringFile` to get that same effect, no? I just had a quick look and found no package using this, so I hope we can remove this (in another PR! This is not a change request for the PR under review!)

Reply:
I guess you mean the `file` option here? Guess it is really old, and yeah, you are probably right it can be done better. So if it comes to me, we can just remove it.

Reply:
And yeah, it seems like the code was added in earlier times by me. It uses `StringFile` though, so that is something ;)